src/components/Carousel/Carousel.tsx

```diff
@@ -74,7 +74,6 @@ export const Carousel: FC<CarouselProps> = React.forwardRef(
       pause = 'hover',
       previousIconButtonAriaLabel: defaultPreviousIconButtonAriaLabel,
       single = false,
-      style,
```

Comment:
should we still keep this? Not sure if there are implementations that are actually using it

Reply:
It's native to the base element, so no need to keep in the props.